### PR TITLE
Allow user to pass in a python boolean to layers.batch_norm

### PR DESCRIPTION
**Pull request by jrock08**
for the training variable.  This should save some computation time during the computation
graph and some memory.  Addresses #30.

### CODE REVIEW

File: aiutils/tftools/batch_normalizer.py [hunk 1]
```diff
@@ -39,9 +39,16 @@ def batchnorm(self, input, training, name, reuse_vars):
...
+            if type(training) == type(True):
+                self.mean, self.variance = tf.nn.moments(input, self.axes)
+                ema_apply_op = self.ema.apply([self.mean, self.variance])
+                self.output = self.get_normalizer(input, training)
+            else:
+                self.output = tf.cond(
+                    training, lambda: self.get_normalizer(input, True),
+                    lambda: self.get_normalizer(input, False))
+
+            print self.output
```
Comment (jrock08) [inline aiutils/tftools/batch_normalizer.py]: Remove debug print statements....

**Comment by jrock08** [comment]
Actually, this appears to be more difficult than I'd realized.  I'm running into issue when I try to load variables.  I'll add a test that catches this, and try to work something out...

**Comment by BigRedT** [comment]
@jrock08 do you need me to review this or are you working on it?

**Comment by jrock08** [comment]
it's broken

**Comment by jrock08** [comment]
I'm closing this for now since I probably won't get to it soon.

**Pull request closed without merging**